### PR TITLE
[flutter_secure_storage] Migrate flutter_secure_storage to Android v2 embedder.

### PR DESCRIPTION
**Pull request by zhenqiu1101**

### CODE REVIEW

**Comment by zhenqiu1101** [comment]
@mehmetf FYI. Could you add reviewer?

File: example/android/gradle.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
 org.gradle.jvmargs=-Xmx1536M
 android.enableJetifier=true
-android.useAndroidX=true
\ No newline at end of file
+android.useAndroidX=true
+android.enableR8=true
```
Comment (mehmetf) [inline example/android/gradle.properties]: We probably don't want to change this.

**Comment by zhenqiu1101** [comment]
@mehmetf Could you merge this PR? I don't have write access.

**Comment by iassal** [comment]
@mehmetf When will this be released?

**Comment by mogol** [comment]
@iassal it has been just published.

**Pull request merged**